content/blog/2018/01/2018-01-26-fosdem-hackaton.adoc

```diff
@@ -0,0 +1,72 @@
+tags:
+- fosdem
+- event
+author: lnewman
+---
+
+On the first weekend in February numerous free and open source developers from around the
+world will travel to Brussels, Belgium, for arguably the largest event of its kind:
+link:https://fosdem.org[FOSDEM].
+Among the thousands of hackers in attendance will be a number of Jenkins contributors.
+
+On the Monday after FOSDEM,
+a group of those contributors will get together to spend a full day hacking on Jenkins and *you* are
+link:https://www.meetup.com/jenkinsmeetup/events/246098584/[invited to join them].
+Folks of all experience levels are welcome,
+there will be sessions for everyone from seasoned hackers to new contributors.
+
+== All-day Jenkins Hackfest
+
+The Hackfest will start at 9:30am on Monday with general introductions and gathering of potential topics/projects.
+Everyone is encouraged to bring topic suggestions that they'd like to work on, but if you don't have one there will be plenty from which to choose.
+The different topic suggestions will all be added to a backlog and we will identify and cluster around popular topics.
+Then people will then split off into smaller groups and work on individual topics in timeboxed sessions.
+
+Meals, snacks, and beverages will be provided throughout the day, wrapping up with dinner around 5pm.
+
+== Something for everyone
+
+Hackfests like this one are a great for contributors of all levels to get invovled, learn from each other, and
+work together on interesting and high impact areas of the project.
+
+Some long-time contributors already know what areas they'll work on and are looking for people interested in joining them.
+Mark Waite (maintainer of the Jenkins Git and Git Client plugins) and Christian Halstrick (SAP) will be spending the day improving the way Git client plugin uses JGit.
+R. Tyler Croy (Jenkins community concierge) and Olivier Vernin (Jenkins infrastructure engineer) will work on infrastructure improvements.
+
+Others contributors, such as Jesse Glick and Andrew Bayer
+(recipients of the  "link:https://jenkins.io/blog/2017/09/08/enumerators-in-pipeline/[A Small Matter of Programming]" award), will arrive without a set plan.
+They will, of course, have some topics to propose, so you might get a chance to work with them.
+Or if you have an area you'd like to work on, they and many other experts will be on hand for discussion and code review.
+
+This is also a great opportunity for new contributors to join the project.
+Baptiste Mathus, long time contributor and all-around nice guy, with host a "New Contributor Hackergarten" covering the basics of contributing to Jenkins and submitting fixes via GitHub Pull requests.
+Even those with minimal coding experience can contribute by improving documentation and making typo fixes via this same process.
+
+== Fun!
+
+More than anything else, Hackfests like this are great fun.
+No matter what your level of exerience, there will be plenty to do and great people to do it with.
```

Comment:
Feels odd to end the sentence with a preposition, but I couldn't find a better way to phrase it.  The article is comfortably informal, and the phrasing matches the article nicely.

Reply:
"... people with whom to do it."